src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/CryptoStream.cs

```diff
@@ -238,7 +238,7 @@ public override void WriteByte(byte value)
         public override int Read(byte[] buffer, int offset, int count)
         {
             CheckReadArguments(buffer, offset, count);
-            return ReadAsyncCore(buffer, offset, count, default(CancellationToken), useAsync: false).ConfigureAwait(false).GetAwaiter().GetResult();
+            return ReadAsyncCore(buffer, offset, count, default(CancellationToken), useAsync: false).GetAwaiter().GetResult();
```

Comment:
Almost everwhere else that had ConfigureAwait(false) kept it.  But here you removed it.

Since you understand it way better than I do, I'll trust you if removing it is right, but I wanted to call it out for confirmation.

Reply:
ConfigureAwait is only relevant when await is being used, as it influences which awaiter is used and thus how its OnCompleted behave.  Here we're just synchronously blocking waiting for the operation to complete, so ConfigureAwait is a nop.